lib/lumberjack/beats/server.rb

```diff
@@ -485,6 +485,14 @@ def read_socket(&block)
       end
     end
 
+    def normalize_v1_metadata_encoding(map)
+      # lets normalize the metadata of the v1 frame to make
+      # sure everything is utf-8 minus the content itself.
+      # We need this to make this plugin backward compatible.
```

Comment:
can you elaborate more on why this is needed? why is the content special in that it can't be force encoded too?